src/mantine-core/src/components/Pagination/demos/controlled.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
```

Comment:
There is no need in controlled demo, it looks and works exactly the same as usage demo, replace it with just a code snippet (see TextInput for reference)